src/encoded/static/components/cart/share.js

```diff
@@ -1,20 +1,30 @@
 /**
  * Components to display the Share Cart modal.
  */
+
+// node_modules
```

Comment:
These should be obvious or am I mistaken...?

Reply:
I’ve started to do this to make it more obvious that some imports are out of order, not that being out of order is a technical issue, but I just prefer not to have library imports below local ones, yet I was doing that sometimes. These comments help me prevent that.

Reply:
Let's talk about this off-line sometimes. Imports cannot be out of order as package management handles that for you. Developers should get an error if they try to do something like call an internal package before an external one.